src/components/Search.js

```diff
@@ -14,38 +13,35 @@ export default function Search({ onSubmit }) {
     fetch(`https://api.rawg.io/api/games?key=${API_KEY}&search=${query}`)
       .then(res => res.json())
       .then(data => setOptions(data.results))
+      .catch(error => console.error(error))
   }, [query])
   return (
-    <section>
+    <>
       <Form onSubmit={handleSubmit}>
         <label>
-          Selected Game: {selectedOption?.name || 'None'}
+          {/* Selected Game: {selectedOption?.name || 'None'} */}
```

Comment:
You could/should delete this comment aswell

Reply:
You are right but I want to keep it as a reminder. I'm not 100% sure yet if I will keep it without the label or if I put it back.